beanstalk_worker/worker_start_scan.py

```diff
@@ -473,13 +516,20 @@ def process_output(self, logs):
             logger.critical(
                 "Failed to run scanners on image under test, moving on!"
             )
-        bs.use("master_tube")
-        job_id = bs.put(json.dumps(scanners_data))
+        else:
+            logger.log(
+                level=logging.INFO,
+                msg=str(scanners_data)
+            )
+        # if weekly scan, push the job for notification
+        if job_info.get("weekly"):
+            bs.use("master_tube")
+            bs.put(json.dumps(job_info))
     except Exception as e:
         logger.fatal(str(e), exc_info=True)
         job_info["action"] = "start_delivery"
         bs.use("master_tube")
-        job_id = bs.put(json.dumps(job_info))
+        bs.put(json.dumps(job_info))
```

Comment:
Is there a reason why we're removing the job info from being logged? It can come handy when debugging issues.